qa/rpc-tests/test_framework/electrumutil.py

```diff
@@ -22,7 +22,7 @@ def bitcoind_electrum_args():
     import random
     global ELECTRUM_PORT
     ELECTRUM_PORT = random.randint(40000, 60000)
-    return ["-electrum=1", "-debug=electrum", "-debug=rpc",
+    return ["-electrum=1", "-electrum.exec=/home/dagurval/electrs/target/release/electrs", "-debug=electrum", "-debug=rpc",
```

Comment:
isn't this path working only if someone has a `dagurval` user defined on his system? :smiley:

Reply:
Oops, fixed